### DISCUSSION TITLE
TFT_Touch/TFT_Touch.cpp

### DISCUSSION THREAD

**Discussion by JayNaire** [Bodmer/TFT_eSPI#2490] [General]
linux->vscode->platformio:
platform = espressif32
board = esp32doit-devkit-v1
framework = arduino
eSPI@2.5.22
In otherwise impressively faultless builds this warning always pops up:
```
.pio/libdeps/esp32doit-devkit-v1/TFT_Touch/TFT_Touch.cpp: In member function 'int16_t TFT_Touch::ReadCal(byte)':
.pio/libdeps/esp32doit-devkit-v1/TFT_Touch/TFT_Touch.cpp:258:1: warning: control reaches end of non-void function [-Wreturn-type]
```
And it's right too. (Not that it causes any problems ... yet!)

**Comment by Bodmer** [comment]
That function is not part of TFT_eSPI however I have updated the TFT_Touch library!

**Reply by JayNaire** [reply]
Thank you.
